Prace_domowe/Praca_domowa1/Grupa4/Hubert_Drążkowski/pd1/Drazkowski_zadanie1.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Pewność siebie to podstawa</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>